devtools/maven/src/main/java/io/quarkus/maven/CreateProjectMojo.java

```diff
@@ -74,6 +74,12 @@
     @Parameter(property = "projectVersion")
     private String projectVersion;
 
+    @Parameter(property = "codestartsEnabled")
+    private Boolean codestartsEnabled;
+
+    @Parameter(property = "withExampleCode")
+    private Boolean withExampleCode;
```

Comment:
What's the difference between them? I thought codestarts were example code?

Reply:
@gastaldi one is temporary, `codestartsEnabled ` is a hack to activate the codestarts codegen until this replace the existing one once and for all.

`withExampleCode` is letting you decide if you want example code or not, example if you want reasteasy, but an empty project (without the ExampleResource)..